src/Reflex/Dom/Builder/Class.hs

```diff
@@ -462,14 +462,20 @@ instance (DomBuilder t m, Monoid w, MonadHold t m, MonadFix m) => DomBuilder t (
 
 -- * Convenience functions
 
+class Unconstrained g h
+instance Unconstrained g h
+
+class DomBuilderSpace a ~ DomBuilderSpace b => SameDomBuilderSpace a b
+instance DomBuilderSpace a ~ DomBuilderSpace b => SameDomBuilderSpace a b
```

Comment:
Makes sense to me; I'll merge this PR now, but please feel free to send one adding that as well.